File: Java/15. Armstrong number in a given range.py
start = int(input("Enter the Starting range of Armstrong:"))
end = int(input("Enter the ending range of Armstrong:"))

for i in range(start, end+1):
    length = len(str(i))
    add = 0
    originalValue = i
    while(originalValue != 0):
        digit = originalValue % 10
        add += pow(digit,length)
        originalValue //= 10

    if(i == add):
        print(i)

File: Java/01. Important Fundamentals/15. Armstrong number in a given range.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the Starting value:");
        int startValue = userInput.nextInt();

        System.out.println("Enter the Ending value:");
        int endValue = userInput.nextInt();

        for( int i=startValue; i<=endValue; i++ ) {
            int length = String.valueOf(i).length();
            int add = 0;
            int originalValue = i;
            
            while(originalValue != 0) {
                int digit = originalValue % 10;
                add += (int) Math.pow(digit,length);
                originalValue /= 10;
            }
            
            if(i == add) {
                System.out.println(i);
            }
        } userInput.close();
    }
}